visualization/README.md

```diff
@@ -140,6 +142,16 @@ $ npm run e2e:auto
 
 > To follow/watch the steps the e2e test is performing, deactivate headless mode in `jest-puppeteer.config.js` (and maybe set the `slowMo` parameter)
 
+## Run in Docker container
+
+You can use this via docker compose or as a standalone container. This section will deal with how to use the visualization as a standalone container. For information on how to use this with docker compose, please check out [Docker Getting Started]({{site.baseurl}}{% link _docs/01-04-docker-containers.md %})
+We assume that you already installed docker, if not, you have to do that before!
+
+To containerize the visualization, please follow the below listed steps.
+
+-   Start the docker container: `docker run -d -p 80:9000 codecharta/codecharta-visualization`. This detaches the container and exposes port 80 on the container and port 9000 on the host.
```

Comment:
This command will provide the visualization localhost:80 on the users machine. I think the port mapping must be turned around.